en/editor/scene/transform-gizmo.md

```diff
@@ -57,4 +57,27 @@ Whe the rect transform tool is active：
 
 In UI elements layout, **rect transform tool** is always used to precisely control the position and length of the node's dimensions. As to the image elements that must keep original image aspect ratio, the rectangle transform tool is normally not the best choice for size adjustment.
```

Comment:
In UI elements layout, **rect transform tool** is always used to precisely control the position and length of the node's dimensions.
-->
In UI elements layout, the **rect transform tool** is always used to precisely control the position and length of the node's dimensions.